packages/dtslint/src/lint.ts

```diff
@@ -44,58 +44,68 @@ export async function lint(
       files.push(fileName);
     }
   }
-  let output = "";
 
+  const options = getEslintOptions(expectOnly, minVersion, maxVersion, tsLocal);
+  const eslint = new ESLint(options);
+  const formatter = await eslint.loadFormatter("stylish");
+  const eresults = await eslint.lintFiles(esfiles);
```

Comment:
looks like this was moved before the esfiles->files rename

Reply:
Yeah, bad merge.